src/ExtensionRoot.ts

```diff
@@ -93,10 +92,7 @@ export class ExtensionRoot {
         // Initialize ExP
         const experimentationService = await createExperimentationServiceAsync(context, new ExperimentationTelemetry(this._logger));
 
-        const expectedCLIVersion: string = await VersionUtility.getExpectedCliVersionAsync(context, experimentationService, packageJsonContent, this._logger);
```

Comment:
💡 This could be done various ways, I was thinking more defensively, we can do this in more smaller chunk to make sure we don't add any side effects like mentioned here: https://github.com/Azure/vscode-bridge-to-kubernetes/issues/4 

we can at first clean these consuming functions and instead of completely removing the Binary utility we can just remove the `v1` code path.